catboost/python-package/catboost/core.py

```diff
@@ -2060,6 +2061,45 @@ def set_params(self, **params):
             self._init_params[key] = value
         return self
 
+    def get_binarized_statistics(self, data, target, feature, prediction_type=None, cat_feature_values=[], plot=False):
```

Comment:
`feature` -> `feature_index`
Also, let's add docstrings to this method like in other methods.

Reply:
Here 'feature' maybe the name of feature, not only index